srcpkgs/PikoPixel.app/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'PikoPixel.app'
+pkgname=PikoPixel.app
+version=1.0.b9e
+revision=1
+archs="x86_64"
+create_wrksrc=yes
+build_wrksrc=PikoPixel.Sources.${version%.*}-${version##*.}/PikoPixel
+build_style=gnu-makefile
```

Comment:
Regarding the `pre_build()` I wonder if the project uses a `./configure` script: if that is the case you can just use `build_style=gnu-configure`

Reply:
it doesn't use a ./configure

Reply:
there is no configure in the source code, it's just helper files and then make :p